core/src/main/java/hudson/util/MultipartFormDataParser.java

```diff
@@ -71,4 +71,23 @@ public void cleanUp() {
         for (FileItem item : byName.values())
             item.delete();
     }
+
+    public static boolean isMultipart(String contentType) {
+        if (contentType == null) {
+            return false;
+        }
+
+        String[] parts = contentType.split(";");
```

Comment:
Why ";"? Many fields and Stapler features use ",". I doubt this method is generic enough